packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -0,0 +1,406 @@
+//! Implementation of display, which can be either fancy or "normal", depending
+//! on the type color and non ascii character can be used.
+//!
+//! This is inspired by [miette](https://docs.rs/miette/latest/miette/index.html)
+
+mod hook;
```

Comment:
No, I thought about it, but(!) even tho `Hooks` is only enabled via the `hooks` feature, the `Hook` trait and the `Builtin` struct, which implements that trait, are always enabled. This means using the same code and logic with `hooks` enabled and disabled. Only that we can modify the behavior using `hooks` instead of only using the builtin (`backtrace` + `spantrace`)

Reply:
Fair, so it's not usable because we require the `once-cell` crate, right? Maybe it's possible to only reexport them in the `fmt` module if the feature is enabled? Then it would be accessible from inside the crate, but as you cannot use it outside of the crate anyway, the user shouldn't use it (to avoid confusion).

Reply:
> Fair, so it's not usable because we require the `once-cell crate`, right?

Yes.

>  Maybe it's possible to only reexport them in the `fmt` module if the feature is enabled? Then it would be accessible from inside the crate, but as you cannot use it outside of the crate anyway, the user shouldn't use it (to avoid confusion).

👍